### ISSUE TITLE
Generates broken background-image with all background properties

### ISSUE THREAD

**Issue by Razunter** [ai/webp-in-css#48]
Plugin is unusable right now, generates broken `background-image` with all `background` properties instead of `url`-only.

Example:
```
h1 {
  background: #fff url("abc.png") no-repeat;
}
```
generates
```
h1 {
  background: #fff url(abc.1ad10566.png) no-repeat;
}
body.no-webp h1,
body.no-js h1 {
  background-image: #fff url(abc.1ad10566.png) no-repeat;
}
body.webp h1 {
  background-image: #fff url(abc.8f924fbb.webp) no-repeat;
}
```

v0.7.0, latest PostCSS

**Comment by ai** [comment]
Please send PR. You need to re-generate `Node#value` here https://github.com/ai/webp-in-css/blob/main/plugin.js#L51-L74

**Comment by Razunter** [comment]
pr: https://github.com/ai/webp-in-css/pull/49

**Comment by ai** [comment]
Fixed in 0.8.
